maintenance_equipment_status/models/maintenance_equipment_status.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields
+
+
+class MaintenanceEquipmentStatus(models.Model):
+    _name = "maintenance.equipment.status"
+    _description = "Maintenance Equipment Status"
+
```

Comment:
_order = 'sequence' is missing here, without it it is very hard to see which is the correct order we want to have